source/class/qx/ui/menu/Menu.js

```diff
@@ -522,11 +522,12 @@ qx.Class.define("qx.ui.menu.Menu",
     // property apply
     _applyOpener : function(value, old)
     {
+      const contentEl = this.getContentElement();
       // ARIA attrs
       if (value) {
-        this.getContentElement().setAttribute("aria-labelledby", value);
-      } else {
-        this.getContentElement().removeAttribute("aria-labelledby");
+        contentEl.setAttribute("aria-labelledby", value);
+      } else if (contentEl) {
```

Comment:
Should'nt this be a general test?

const contentEl = this.getContentElement();
if  (contentEl) {

...
}

Reply:
I don't see a reason why the content element should be null or undefined in the first if case. That goes for all the places i added the check or did not.

Reply:
I would probably have added, at line 525.5, something like,

```
if (! contentEl) {
  return;
}

That handles the case where the content element doesn't exist right away (we won't do anything). In that case, line 529 could get rid of the `if` constraint.

I agree, as is it's correct but a bit hard to read.
```